packages/terra-functional-testing/src/reporters/spec-reporter/merge-results.js

```diff
@@ -34,7 +34,7 @@ const getResultsFileName = (options) => {
     theme,
   } = options;
 
-  const keys = [locale, theme, formFactor, browser, packageName].filter(key => !!key);
+  const keys = [locale, theme, formFactor, browser, packageName.split('/').pop()].filter(key => !!key);
```

Comment:
Could we add a test for this change?

Reply:
Added a change here, mostly just copied the original test but changed the package name to be a scoped one
https://github.com/cerner/terra-toolkit/pull/617/commits/6b8f2b8d59fce9ff47c70329be39d2968fb4c3cf#diff-8da0fd05408774e858c8ab388acd14d3ed93b9387e33752fb3031dfe23361764R47